index.js

```diff
@@ -156,7 +156,7 @@ function writeP(filePath, content) {
 
   return writeFile(filePath, content).catch(function(reason) {
     if (reason && reason.code === 'ENOENT') {
-      return mkdirp(base).then(function() {
+      return mkdirp(base, { mode: '0775' }).then(function() {
```

Comment:
No, you need write+execute on directories if you actually want to modify the contents.  See  http://www.hackinglinuxexposed.com/articles/20030424.html